python/cuvs/cuvs/neighbors/cagra/cagra.pyx

```diff
@@ -177,7 +189,10 @@ cdef class IndexParams:
                  graph_degree=64,
                  build_algo="ivf_pq",
                  nn_descent_niter=20,
-                 compression=None):
+                 compression=None,
+                 ivf_pq_build_params: ivf_pq.IndexParams = None,
```

Comment:
I like the type annotations! We should do this everywhere =)  (but not for the 25.06 release)